feature/gribi/otg_tests/base_hierarchical_nhg_update/base_hierarchical_nhg_update_test.go

```diff
@@ -290,25 +292,64 @@ func gribiClient(ctx context.Context, t *testing.T, dut *ondatra.DUTDevice) (*fl
 // 100% loss.
 //
 // TODO: Packets should be validated to arrive at ATE with destination MAC pMAC.
-func validateTrafficFlows(t *testing.T, ate *ondatra.ATEDevice, good []*ondatra.Flow, bad []*ondatra.Flow) {
-	if len(good) == 0 && len(bad) == 0 {
-		return
-	}
+func validateTrafficFlows(t *testing.T, ate *ondatra.ATEDevice, goodFlow, badFlow string) {
 
-	flows := append(good, bad...)
-	ate.Traffic().Start(t, flows...)
+	config := ate.OTG().FetchConfig(t)
+	waitOTGARPEntry(t)
```

Comment:
Move this to be a single check in the TestBaseHierarchicalNHGUpdate function? We should only need to check this once and then it's clear that it's not part of traffic validation.

Reply:
Moved